detekt.yml

```diff
@@ -841,6 +842,7 @@ datadog:
       - "kotlin.collections.MutableSet.joinToString(kotlin.CharSequence, kotlin.CharSequence, kotlin.CharSequence, kotlin.Int, kotlin.CharSequence, kotlin.Function1?)"
       - "kotlin.sequences.Sequence.filter(kotlin.Function1)"
       - "kotlin.sequences.Sequence.forEach(kotlin.Function1)"
+      - "java.util.LinkedHashMap.remove(kotlin.String)"
```

Comment:
nit: this should go before `kotlin` entries

Reply:
Oh...my bad tks !!